### PR TITLE
Implementing geolexica site for 204

### LINKED ISSUE #1: Implement Geolexica site for ISO 14812
The terminology is available at:
* https://github.com/ISO-TC204/iso14812
* https://github.com/metanorma/iso-14812/

### PR DESCRIPTION
**Pull request by HassanAkbar**
Implementing geolexica site for 204

closes #1

### CODE REVIEW

**Comment by ronaldtse** [comment]
@HassanAkbar please remember to bump version for geolexica-server and jekyll-geolexica once you're done, thanks!

File: isotc204-glossary/tc204-termbase.meta.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+header:
+  register-name: ISO/TC 204 Intelligent transport systems (ITS)
+  content-summary: A compound register containing terminology records that have been
+    published in ISO/TC 204-sponsored International Standards and Technical Specifications.
+  date-of-last-change: '2022-05-30'
```
Comment (HassanAkbar) [inline isotc204-glossary/tc204-termbase.meta.yaml:6]: This is the date when [iso-14812 source XML file](https://github.com/metanorma/iso-14812/blob/main/reference-docs/sts/iso_std_iso_ts_14812_ed-1_v1_en.xml) is last updated. 

@ronaldtse Is this correct or do I need to use some other date?

Comment (ronaldtse) [reply isotc204-glossary/tc204-termbase.meta.yaml:6]: Yes this is correct.

**Comment by ronaldtse** [comment]
Thank you @HassanAkbar !

**Pull request merged**